src/test/java/ladder/domain/BarsTest.java

```diff
@@ -0,0 +1,50 @@
+package ladder.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class BarsTest {
+
+    @Test
+    void 주어진_개수만큼_좌표값을_생성한다() {
+        Bars bars = new Bars(new RandomPointGenerator(), 5);
+
+        assertThat(bars.toUnmodifiableBars()).hasSize(5);
+    }
+
+    @Test
+    void 좌표값은_랜덤하게_생성된다() {
+
+        List<Boolean> expected = List.of(true, false, true, false, true);
+
+        Queue<Boolean> queue = new LinkedList<>(expected);
+
+        Bars bars = new Bars(new MockedPointGenerator(queue), 5);
+
+        assertThat(bars.toUnmodifiableBars()).containsExactly(true, false, true, false, true);
```

Comment:
```suggestion
        assertThat(bars.toUnmodifiableBars()).isEqualTo(expected);
```

똑같은 값으로 보이는데 리스트를 재사용하지 않으신 이유가 있을까요?

Reply:
적절한 메소드가 containsExactly밖에 생각나지 않아 해당 메소드를 사용하려고 했는데 list를 인자로 받을 수 없어 고민하다가 위와같이 작성했습니다.
isEqualTo로 비교가되는지 몰랐네요 😅